ECOSYSTEM.md

```diff
@@ -38,9 +47,15 @@ Core contributors are individuals who contribute to the React Native project. A
 
 These are behaviors we have observed in our existing core contributors. They aren't strict rules but rather outline their usual responsibilities. We do not expect every core contributor to do all of the above things all the time. Most importantly, we want to create a supportive and friendly environment that fosters collaboration. Above all else, **we are always polite and friendly.**
 
-Core contributor status is attained after consistently contributing and taking on the responsibilities outlined above and granted by other core contributors. Similarly, after a long period of inactivity, a core contributor may be removed.
+Core contributor status is attained after consistently contributing and taking on the responsibilities outlined above and granted by other core contributors. Similarly, after a long period of inactivity (~6 months or more), a core contributor may be contacted to understand if they’re still interested in being part of the program.
+
+You can use this form <MISSING_LINK_TODO> to either:
```

Comment:
really like the idea of having a form for people to become core contributors 👍

Reply:
@cortinico's idea!